File: LEARN/TestApp/TestApp/ViewController.m
//
//  ViewController.m
//  TestApp
//
//  Created by Ivo Stoyanov on 1/27/15.
//  Copyright (c) 2015 s1nn. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
- (IBAction)onCLick:(id)sender;
@property (weak, nonatomic) IBOutlet UILabel *label;

@end

@implementation ViewController

int clickCount = 0;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)onCLick:(id)sender {
    clickCount++;
    self.label.text = [NSString stringWithFormat: @"Clicked: %d times", clickCount];
    [self.label sizeToFit];
}
@end

File: LEARN/BullsEye/BullsEye/ViewController.swift
//
//  ViewController.swift
//  BullsEye
//
//  Created by Ivo Stoyanov on 10/15/14.
//  Copyright (c) 2014 HOME. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var currentValue = 50
    
    @IBOutlet weak var slider: UISlider!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        currentValue = lroundf(slider.value)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func showAlert() {
        let message = "The value: \(currentValue)"
        let alert = UIAlertController(title: "Value", message: message, preferredStyle: .Alert)
        let action = UIAlertAction(title: "OK", style: .Default, handler: nil)
        alert.addAction(action)
        presentViewController(alert, animated: true, completion: nil)
    }

    
    @IBAction func sliderMoved(slider: UISlider) {
        currentValue = lroundf(slider.value)
        //println("Slider: \(currentValue)")
    }
    
}

